guides/common/modules/proc_configuring-keycloak-settings-for-authentication-with-cac-cards.adoc

```diff
@@ -1,23 +1,41 @@
 [id="configuring-keycloak-settings-for-authentication-with-cac-cards_{context}"]
 = Configuring {Keycloak} settings for authentication with {PIV} cards
 
-You must configure {Keycloak} settings for authentication with {PIV} cards.
+If you want usres to authenticate with {PIV} cards, configure {Keycloak} for authentication with {PIV} cards and configure the operating system for {PIV} cards on each user system.
 
 .Procedure
 
-. In the {Keycloak} web UI, navigate to the *Authentication* tab.
+In the {Keycloak} web UI:
+
+. Navigate to the *Authentication* tab.
 . From the *Flows* list, select *Browser*.
-. Click *Copy* to copy this flow.
-. In the *Copy Authentication Flow* window, enter a new name for the flow and click *OK*.
-. In the copied flow, delete *Username Password Form* and *OTP Form* entries.
-. Click *Add execution*.
-. From the *Provider* list, select *X509/Validate Username Form*.
-. Click *Save*.
-. In the *X509/Validate Username Form* raw, select *ALTERNATIVE*.
-. In the *X509/Validate Username Form* raw, click *Actions* > *Config*.
-. In the *Alias* field, enter a name for this configuration.
-. From the *User Identity Source* list, select *Subject’s Common Name*,
-. From the *User mapping method* list, select *Username or Email*.
-. Click *Save*.
+.. Click *Copy* to copy this flow.
+.. In the *Copy Authentication Flow* window, enter a new name for the flow and click *OK*.
+.. In the copied flow, delete the *Username Password Form* and *OTP Form* entries.
+.. Click *Add execution*.
+.. From the *Provider* list, select *X509/Validate Username Form*.
+.. Click *Save*.
+. In the *X509/Validate Username Form* row, select *ALTERNATIVE*.
+Click *Actions* > *Config*.
+.. In the *Alias* field, enter a name for this configuration.
+.. From the *User Identity Source* list, select *Subject’s Common Name*,
+.. From the *User mapping method* list, select *Username or Email*.
+.. Click *Save*.
 . Navigate to *Authentication* > *Bindings*.
 . From the *Browser Flow* list, select the created flow.
+
+On each system from which you want users to log in to with {PIV} cards:
+
+. Install the required package:
++
+[options="nowrap", subs="+quotes,attributes"]
+----
+# {project-package-install} opensc
```

Comment:
It sounds like this is on non-Foreman servers so `dnf install` or equivalent is probably what you want.